integrations/operator/Makefile

```diff
@@ -98,10 +98,8 @@ manifests: crdgen controller-gen ## Generate WebhookConfiguration, ClusterRole a
 			"$${proto}"; \
 	done
 
-# Some comments have `{{xyz}}` which triggers the golang templating engine when installing the chart.
-# We don't want that, so we escape them.
-	$(foreach file, $(wildcard $(CURDIR)/config/crd/bases/*), \
-		$(shell sed 's/`{{/{{ `{{/g;s/}}`/}}` }}/g' $(file) > $(CURDIR)/../../examples/chart/teleport-cluster/charts/teleport-operator/templates/$(notdir $(file))))
+	rm -r ../../examples/chart/teleport-cluster/charts/teleport-operator/operator-crds/
```

Comment:
maybe we should move this into a bash script

Reply:
Before the PR it should definitely have been a bash script, but now I think it's fine as we're just doing rm -r and cp -r.